### PR TITLE
Merge release-1.18 to master after v1.18.1 release

### PR DESCRIPTION
**Pull request by jchen6585**
**What type of PR is this?**

release workflow

**Which issue does this PR fix?**:

N/A

**What does this PR do / Why do we need it?**:
Fixes merge conflicts from release-1.18 to main

```
git checkout master
git merge github/release-1.18
fix merge conflicts
```

**Testing done on this change**:

**Will this PR introduce any new dependencies?**:

**Will this break upgrades or downgrades? Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jaydeokar — DISMISSED** [review]
lgtm

File: charts/cni-metrics-helper/README.md [hunk 1]
```diff
@@ -55,34 +55,26 @@ $ helm uninstall cni-metrics-helper --namespace kube-system
...
-| `containerSecurityContext`     | SecurityContext to set on the container                       | `{}`                                |
-| `tolerations`                  | Optional deployment tolerations                               | `[]`                                |
-| `updateStrategy`               | Optional update strategy                                      | `{}`                                |
-| `imagePullSecrets`             | Docker registry pull secret                                   | `[]`                                |
-| `nodeSelector`                 | Node labels for pod assignment                                | `{}`                                |
-| `tolerations`                  | Optional deployment tolerations                               | `[]`                                |
-
+| Parameter                    | Description                                                   | Default            |
+|------------------------------|---------------------------------------------------------------|--------------------|
+| fullnameOverride             | Override the fullname of the chart                            | cni-metrics-helper |
```
Comment (jaydeokar) [inline charts/cni-metrics-helper/README.md:60]: Is this an intentional update ?

Comment (jchen6585) [reply charts/cni-metrics-helper/README.md]: Fixed

**Review by jaydeokar — APPROVED** [review]
lgtm

**Pull request merged**